ui/components/app/collectibles-items/collectibles-items.component.js

```diff
@@ -0,0 +1,149 @@
+
+            return (
+              <div key={`collection-${index}`}>
+                <Box
+                  marginTop={4}
+                  marginBottom={4}
+                  display={DISPLAY.FLEX}
+                  alignItems={ALIGN_ITEMS.CENTER}
+                  justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+                >
+                  <Box alignItems={ALIGN_ITEMS.CENTER}>
+                    <img width="28" src={icon} />
+                    <Typography
+                      color={COLORS.BLACK}
+                      variant={TYPOGRAPHY.H4}
+                      margin={[0, 0, 0, 2]}
+                    >
+                      {`${key} (${collectibles.length})`}
+                    </Typography>
+                  </Box>
+                  <Box alignItems={ALIGN_ITEMS.FLEX_END}>
+                    <i
+                      className={`fa fa-lg fa-chevron-${
+                        isExpanded ? 'down' : 'right'
+                      }`}
+                      onClick={() => {
+                        setDropdownState((_dropdownState) => ({
+                          ..._dropdownState,
+                          [key]: !isExpanded,
+                        }));
+                      }}
+                    />
+                  </Box>
+                </Box>
+                {isExpanded ? (
+                  <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP}>
+                    {collectibles.map((collectible, i) => {
+                      const width =
+                        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
+                          ? BLOCK_SIZES.ONE_THIRD
+                          : BLOCK_SIZES.ONE_SIXTH;
+                      return (
+                        <Box width={width} padding={2} key={`collectible-${i}`}>
+                          <Box
+                            borderRadius={SIZES.MD}
+                            backgroundColor={collectible.backgroundColor}
+                          >
+                            <img src={collectible.icon} />
+                          </Box>
+                        </Box>
+                      );
+                    })}
+                  </Box>
+                ) : null}
+              </div>
+            );
+          })}
+          <Box
+            marginTop={6}
+            flexDirection={FLEX_DIRECTION.COLUMN}
+            justifyContent={JUSTIFY_CONTENT.CENTER}
+          >
+            <Typography
+              color={COLORS.UI3}
+              variant={TYPOGRAPHY.H5}
+              align={TEXT_ALIGN.CENTER}
+            >
+              {t('missingNFT')}
+            </Typography>
+            <Box
+              alignItems={ALIGN_ITEMS.CENTER}
+              justifyContent={JUSTIFY_CONTENT.CENTER}
+            >
+              <Box justifyContent={JUSTIFY_CONTENT.FLEX_END}>
+                <Button
+                  type="link"
+                  onClick={onRefreshList}
+                  style={{ padding: '5px' }}
```

Comment:
I see PR uses a couple of different ways to style like inline styles here `style={{ padding: '5px' }}`, and design guideline is to use dimensions as 4, 8, 16, 24,... which is also not followed.